### PR TITLE
Metadata updates

### PR DESCRIPTION
**Pull request by ZaraGlynn**

### CODE REVIEW

File: src/admin/items/update.ts [hunk 1]
```diff
@@ -62,24 +29,82 @@ export const updateByS3key = async (event: APIGatewayProxyEvent): Promise<APIGat
...
+        record_label: Joi.string(),
+        series_name: Joi.string(),
+        episode_name: Joi.string(),
+        episode_number: Joi.boolean(),
+        recording_name: Joi.string(),
+        speakers: Joi.array().items(Joi.string()),
+        performers: Joi.array().items(Joi.string()),
+        host_organization: Joi.array().items(Joi.string()),
+        radio_station: Joi.string(),
+        other_metadata: JSON,
```
Comment (matthewberryman) [inline src/admin/items/update.ts]: pretty sure we want `Joi.object()` here

File: src/admin/items/update.ts [hunk 2]
```diff
@@ -62,24 +29,82 @@ export const updateByS3key = async (event: APIGatewayProxyEvent): Promise<APIGat
...
+        item_name: Joi.string(),
+        original_title: Joi.string(),
+        related_event: Joi.string(),
+        volume_in_series: Joi.boolean(),
```
Comment (matthewberryman) [inline src/admin/items/update.ts]: @ZaraGlynn should be Joi.number().integer()

File: src/admin/items/update.ts [hunk 3]
```diff
@@ -62,24 +29,82 @@ export const updateByS3key = async (event: APIGatewayProxyEvent): Promise<APIGat
...
         directors: Joi.array().items(Joi.string()),
         writers: Joi.array().items(Joi.string()),
-        collaborators: Joi.string(),
+        collaborators: Joi.array().items(Joi.string()),
         exhibited_at: Joi.string(),
         series: Joi.string(),
         ISBN: Joi.number().integer(),
+        DOI: Joi.string(),
         edition: Joi.number().integer(),
+        year_produced: Joi.number().integer(),
```
Comment (Dan-Wood) [inline src/admin/items/update.ts]: Remove this, can get the year from Time Produced.

File: src/admin/items/update.ts [hunk 4]
```diff
@@ -62,24 +29,82 @@ export const updateByS3key = async (event: APIGatewayProxyEvent): Promise<APIGat
...
+        original_title: Joi.string(),
+        related_event: Joi.string(),
+        volume_in_series: Joi.boolean(),
+        organisation: Joi.string(),
+        OA_highlight: Joi.boolean(),
+        TBA21_material: Joi.boolean(),
+        OA_original: Joi.boolean(),
+        lecturer: Joi.string(),
+        authors: Joi.array().items(Joi.string()),
+        credit: Joi.string(),
```
Comment (Dan-Wood) [inline src/admin/items/update.ts]: Meta spreadsheet says this is required.

File: schema/schema.sql [hunk 5]
```diff
@@ -61,27 +137,92 @@ CREATE TABLE tba21.collections
...
+  focus_scitech numeric(1),
+  url varchar(2048),
+  related_material bigint[],
+  license tba21.licence_type,
+  location varchar(256),
+  other_metadata jsonb
+);
+
+--Contributor metadata
+CREATE TABLE tba21.profile
```
Comment (mehrdadgit) [inline schema/schema.sql]: Is this user profile  ? if so why not make single (not array) contributor uuid as pkey ? @matthewberryman

Comment (matthewberryman) [reply schema/schema.sql]: multiple people can work on the same collection

Comment (mehrdadgit) [reply schema/schema.sql]: but I see a contributor[] on collection table as well...unless you mean a 'group' of user share 'one' profile...

Comment (matthewberryman) [reply schema/schema.sql]: Sorry, I thought you were talking about collection table as your comment was under the diff showing the change there, and not showing profile. Yes profile should just be singular.

Comment (matthewberryman) [reply schema/schema.sql]: ^^ @ZaraGlynn please update schema/seeds/joi for profile table, in fact shouldn't even need a contributor singular at all. The ID is the contributor ID for a profile.

File: src/admin/collections/insert.ts [hunk 6]
```diff
@@ -40,7 +40,7 @@ export const createCollection = async (event: APIGatewayProxyEvent): Promise<API
         publisher: Joi.array().items(Joi.string()),
         interviewers: Joi.array().items(Joi.string()),
         interviewees: Joi.array().items(Joi.string()),
-        cast_: Joi.string(),
+        cast_: Joi.array().items(Joi.string()),
         title: Joi.string(),
```
Comment (Dan-Wood) [inline src/admin/collections/insert.ts]: Make sure the required fields are set as JOI required too, please.

**Pull request merged**